bootstrap/components/sources/kafka-source-topology-component.json

```diff
@@ -35,13 +35,22 @@
         "hint": "schema,override"
       },
       {
-        "uiName": "Reader schema version",
+        "uiName": "Reader Schema Branch",
+        "fieldName": "schemaBranch",
```

Comment:
Same comment for isOptional field as i made in kafka sink

Reply:
yes, need to add `isOptional` field with false.